### PR TITLE
feat: display quantity for checkout if no identifierinput

### PR DESCRIPTION
**Pull request by ddwsd**
Displays quantity for checkout if identifierInput does not exist. This is mainly for the FFTH distribution. It will affect those distributions of type "REDEEM".

### CODE REVIEW

File: src/components/CustomerQuota/CheckoutSuccess/RedeemedItem.tsx [hunk 1]
```diff
@@ -12,26 +12,28 @@ export const RedeemedItem: FunctionComponent<{
...
+
+  const quantitiesDisplay = Object.values(quantities)[0];
 
   return (
     <View style={{ marginBottom: size(1.5) }}>
       <View style={sharedStyles.itemRow}>
         <AppText style={sharedStyles.itemHeaderText}>{categoryName}</AppText>
       </View>
-      {!!identifierInputDisplay && (
+      {
```
Comment (sebastianquek) [inline src/components/CustomerQuota/CheckoutSuccess/RedeemedItem.tsx:29]: Do we still need these parentheses?

Comment (ddwsd) [reply src/components/CustomerQuota/CheckoutSuccess/RedeemedItem.tsx:29]: oh yeah we dont oops

File: src/components/CustomerQuota/CheckoutSuccess/RedeemedItem.tsx [hunk 2]
```diff
@@ -12,26 +12,28 @@ export const RedeemedItem: FunctionComponent<{
...
-  const { category, identifierInputs } = itemQuantities;
+  const { category, identifierInputs, quantities } = itemQuantities;
   const categoryName = getProduct(category)?.name ?? category;
 
   const identifierInputDisplay = identifierInputs
     ? getIdentifierInputDisplay(identifierInputs)
-    : "";
+    : null;
+
+  const quantitiesDisplay = Object.values(quantities)[0];
```
Comment (ilhaoni) [inline src/components/CustomerQuota/CheckoutSuccess/RedeemedItem.tsx:22]: Just wondering, will there be an edge case where quantities is null, or empty? Will this fail? :O

Comment (ddwsd) [reply src/components/CustomerQuota/CheckoutSuccess/RedeemedItem.tsx:22]: based on the types, no

**Pull request merged**